LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ILoraModuleClient.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer.BasicsStation.ModuleConnection
+{
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Shared;
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface ILoraModuleClient : IAsyncDisposable
+    {
+        /// <summary>
+        /// Operation Timeout for the module connection in milliseconds.
```

Comment:
I don't think you need to mention the unit since a `TimeSpan` converts automatically between several of those:

```suggestion
        /// Operation timeout for the module connection.
```